db/seeds/fill.rb

```diff
@@ -37,31 +39,21 @@
   end
 end
 
-Rails.logger.debug 'Seeding requests..'
-requests = FactoryBot.build_list(:request, request_count) do |request|
-  request.class.skip_callback(:create, :after, :broadcast_request, raise: false)
-  request.save!
-end
-
-Rails.logger.debug 'Seeding contributors..'
-contributors = FactoryBot.create_list(:contributor, contributors_count)
-
 FactoryBot.modify do
   factory :message do
-    created_at { message_time }
-    updated_at { message_time }
     sender_type { 'Contributor' }
     text { Faker::Lorem.paragraph }
     unknown_content { false }
     broadcasted { false }
     sender { contributors.sample }
     recipient { nil }
-    request { requests.sample }
   end
 end
 
-Rails.logger.debug 'Seeding messages..'
-FactoryBot.build_list(:message, message_count) do |message|
-  message.class.skip_callback(:create, :after, :send_if_outbound, raise: false)
-  message.save!
+Rails.logger.debug 'Seeding requests..'
+FactoryBot.build_list(:request, request_count) do |request|
+  Message.skip_callback(:commit, :after, :send_if_outbound, raise: false)
+  request.save!
+  Rails.logger.debug 'Seeding requests replies...'
+  FactoryBot.create_list(:message, replies_count, request: request)
```

Comment:
I can of course also test this locally, I've held off on approving this since this would make a big impact for me as I have actually production values in my `.env` setup that could cost us money if the seeds were run and the callbacks were not properly skipped.